packages/vision/src/helpers.js

```diff
@@ -73,14 +73,27 @@ var coerceImage = (image, callback) => {
  * @return {Function} - The function that, when called, will call annotateImage
  *   asking for the single feature annotation.
  */
-var _createSingleFeatureMethod = featureValue => {
-  return function(image, options) {
-    return this.annotateImage({
-      image: image,
-      features: [{type: featureValue}],
-    }, options);
-  };
-};
+ var _createSingleFeatureMethod = featureValue => {
+   return function(annotateImageRequest, callOptions) {
+     annotateImageRequest.features = annotateImageRequest.features || [];
+
+     // Ensure the feature value indicated by the user's method choice exists on the
+     // features array; if it does not, add it.
+     var found = false;
```

Comment:
I differentiate between "unusual" and "incorrect". I think the way to handle it is do what we say we're going to do, and return crop hints from cropHints() and nothing else. I don't think any of our API methods should have a quirk where they start working as "extra-powered" if the user accidentally unlocks it. We shouldn't enable that with our time of development, maintenance, or support for it. The code should just do what our docs say it does.